### DISCUSSION TITLE
log_size_limit missing

### DISCUSSION THREAD

**Discussion by giomke** [Security-Onion-Solutions/securityonion#11684] [2.4] [answered]
### Version

2.4.10

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

8

### RAM

32

### Storage for /

500

### Storage for /nsm

500

### Network Traffic Collection

span port

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

No, there are no additional clues

### Detail

Unfortunately, I can't find the way how to check and change log_size_limit value in a new security onion.
Is it removed from the new 2.4 release

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by dougburks** [comment, marked as answer]
Please see https://docs.securityonion.net/en/2.4/elasticsearch.html#deleting-indices.
